src/makeRobotAccountant.js

```diff
@@ -1,12 +1,24 @@
-'use strict';
+"use strict";
 
 /**
  *
  * @return {function}
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let cashCount = 0;
+
+  return (a) => {
+    return (b) => {
+      ++cashCount;
```

Comment:
If you use increment, use postfix.
As a separate expression it doesn't really matter, but it's more readable.
```suggestion
      cashCount++;
```